csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
To be honest, I'm not sure how the .csproj and project.json interoperate.
For gRPC I have two different solutions - one contains only .csproj files and the other one contains only project.json files.  That way, all the relevant settings are set in the project.json files and it is easy to see what options get applied and which do not.

Also, from the project.json file you should be able to obtain the nuget package with "dotnet pack".